src/mcpdotnet/Protocol/Types/Tool.cs

```diff
@@ -9,18 +11,18 @@ public class Tool
     /// <summary>
     /// The name of the tool.
     /// </summary>
-    [System.Text.Json.Serialization.JsonPropertyName("name")]
+    [JsonPropertyName("name")]
     public string Name { get; set; } = string.Empty;
```

Comment:
Should we mark as required to avoid accidentally emitting empty strings?

Reply:
Same here